src/Orleans.Runtime/Hosting/DefaultSiloServices.cs

```diff
@@ -371,10 +371,7 @@ internal static void AddDefaultServices(IServiceCollection services)
             services.AddSingleton<IPostConfigureOptions<OrleansJsonSerializerOptions>, ConfigureOrleansJsonSerializerOptions>();
             services.AddSingleton<OrleansJsonSerializer>();
 
-            services.TryAddTransient<IMessageSerializer>(sp => ActivatorUtilities.CreateInstance<MessageSerializer>(
-                sp,
-                sp.GetRequiredService<IOptions<ClientMessagingOptions>>().Value.MaxMessageHeaderSize,
-                sp.GetRequiredService<IOptions<ClientMessagingOptions>>().Value.MaxMessageBodySize));
```

Comment:
This is actually a mistake, it should be `SiloMessagingOptions` here

Reply:
Fixed it in the other PR.